backend/tests/app/routes/test_suggestions.py

```diff
@@ -0,0 +1,101 @@
+# pylint: disable=missing-module-docstring,missing-function-docstring
+from unittest.mock import ANY, AsyncMock
+
+import pytest
+from fastapi import FastAPI
+from httpx import AsyncClient
+from starlette import status
+
+URL = "web:post-search-suggestions"
```

Comment:
First, this is not an `URL` - calling `app.url_path_for(URL)` is a little bit confusing.
Secondly it is a little bit contentious topic, but I believe, that it's better to specify URLs for tests without using resolver (`app.url_path_for`) In the end URL is part of the contract, so it should be defined and unchanging - the same as result and payload. Doing it this way will de facto exclude it from tests.

This is the standard of "do as I say, not as I do" as I understand that all of our tests do this. But it's more a convention that should change.

We can decide to retain `app.url_path_for` for now, but at least changing the name of the variable from `URL` to `SEARCH_SUGGESTION_PATH` would be advisable.